psalm.xml.dist

```diff
@@ -4,6 +4,9 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    findUnusedPsalmSuppress="true"
+    findUnusedBaselineEntry="true"
+    findUnusedCode="false"
```

Comment:
Should this not be `true` so it acts as a signal for uncovered methods and dead code? Issues with PHPUnit data providers can be baselined until the PHPUnit psalm plugin catches up…

Reply:
Libraries have unused code. ...unless psalm considers test code a usage.

Reply:
test code is code usage

Reply:
With enabled unused code psalm reports unused to/from Zend methods. I am inclined to just baseline them.
https://github.com/laminas/laminas-psr7bridge/blob/e8ab789d37a4651e8441d0be69b6c02fd229fcc2/src/Psr7ServerRequest.php#L154-L170

Reply:
Yes, baselining them is fine :+1: